dom0/sd-clean-all.sls

```diff
@@ -10,6 +10,17 @@ set-fedora-as-default-dispvm:
 include:
   - sd-usb-autoattach-remove
 
+sd-cleanup-whonix-gw-15:
+  cmd.run:
+    - names:
+      - qvm-run whonix-gw-15 'sudo rm -f /etc/rsyslog.d/sdlog.conf'
+      - qvm-run whonix-gw-15 'sudo rm -f /etc/apt/sources.list.d/securedrop_workstation.list'
+      - qvm-run whonix-gw-15 'sudo apt remove -y securedrop-log'
+      - qvm-run whonix-gw-15 'sudo systemctl restart rsyslog'
+      - qvm-run whonix-gw-15 'sudo apt-key del 4ED79CC3362D7D12837046024A3BE4A92211B03C'
```

Comment:
We should also remove the prod key here:  "22245C81E3BAEB4138B36061310F561200F4AD77" (the command will always return zero so we can run both even in dev, and not worry about split logic in the cleanup action.

Reply:
Okay, i will add that too.

Reply:
The `apt remove -y securedrop-log` line is not idempotent. So far, we've been careful to ensure that `make clean` exits zero even if run multiple times.